package.json

```diff
@@ -0,0 +1,98 @@
+{
+  "name": "project-name",
+  "version": "0.1.0",
+  "description": "The https://{project-domain}.com web app",
+  "author": "MOXY <hello@moxy.studio>",
+  "private": true,
+  "license": "MIT",
+  "keywords": [
+    "www",
+    "next"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:{project-org}/{project-name}"
+  },
+  "scripts": {
+    "dev": "node server",
+    "prebuild": "rimraf .next",
+    "build": "next build",
+    "export": "next export",
+    "start": "cross-env NODE_ENV=production node server",
+    "test": "jest",
+    "coverage-report": "open coverage/lcov-report/index.html",
+    "lint": "npm run lint:eslint && npm run lint:stylelint",
+    "lint:eslint": "eslint --ignore-path .gitignore .",
+    "lint:stylelint": "stylelint --ignore-path .gitignore \"**/*.css\""
+  },
+  "dependencies": {
+    "compression": "^1.7.4",
+    "cross-env": "^5.2.0",
+    "dotenv": "^8.1.0",
+    "express": "^4.17.1",
+    "express-static-gzip": "^1.1.3",
+    "keyboard-only-outlines": "^1.0.4",
+    "lodash": "^4.17.15",
+    "next": "^9.0.0",
+    "normalize.css": "^8.0.1",
+    "prop-types": "^15.7.2",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-ga": "^2.6.0"
+  },
+  "devDependencies": {
+    "@commitlint/config-conventional": "^8.0.0",
+    "@zeit/next-css": "^0.2.1-canary.4",
+    "babel-jest": "^24.8.0",
+    "babel-preset-moxy": "^3.1.0",
+    "commitlint": "^8.0.0",
+    "compression-webpack-plugin": "^3.0.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.13.2",
+    "eslint": "^6.0.1",
+    "eslint-config-moxy": "^7.1.0",
+    "file-loader": "^4.0.0",
+    "husky": "^3.0.0",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^24.8.0",
+    "jest-enzyme": "^7.0.2",
+    "jest-file": "^1.0.0",
+    "lint-staged": "^9.1.0",
+    "mime-db": "^1.40.0",
+    "next-compose-plugins": "^2.2.0",
+    "postcss-preset-moxy": "^4.3.0",
+    "raw-loader": "^3.0.0",
+    "rimraf": "^2.6.3",
+    "standard-version": "^6.0.1",
+    "stylelint": "^10.1.0",
+    "stylelint-config-moxy": "^6.2.0",
+    "svg-css-modules-loader": "^1.6.1",
+    "svgo-loader": "^2.2.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+    }
+  },
+  "standard-version": {
+    "scripts": {
+      "posttag": "git push --follow-tags origin master"
```

Comment:
I believe it's more obvious to have a _npm script_ (such as `npm run release`) to run `standard-version` as it seems the recommended way by [standard-version](https://github.com/conventional-changelog/standard-version#as-npm-run-script) and have a `postrelease` _script_ to push the tag, removing the need for this object to exist.

Reply:
Yes we should have a release npm script that runs standard-version. I’m also in favor of having a postrelease script instead of having a separate key in here.